vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/start_retirement.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Description: This method sets the retirement_state to retiring
+#
+
+$evm.log("info", "Listing Root Object Attributes:")
+$evm.root.attributes.sort.each { |k, v| $evm.log("info", "\t#{k}: #{v}")  }
+$evm.log("info", "===========================================")
+
+service = $evm.root['service']
+if service.nil?
+  $evm.log('info', "Service Object not found")
+  exit MIQ_ABORT
+end
+
+$evm.log('info', "Service before start_retirement: #{service.inspect} ")
+
+if service.retired?
+  $evm.log('info', "Service is already retired. Aborting current State Machine.")
+  exit MIQ_ABORT
+end
+
+if service.retiring?
+  $evm.log('info', "Service is in the process of being retired. Aborting current State Machine.")
+  exit MIQ_ABORT
```

Comment:
Use `fatal` logging messages here if we are exiting the state machine with `MIQ_ABORT`.